rootdir/vendor/etc/fstab.seine

```diff
@@ -5,7 +5,7 @@ system                                                  /system
 product                                                 /product                  ext4    ro,barrier=1,discard                                 wait,slotselect,avb=vbmeta_system,logical,first_stage_mount
 vendor                                                  /vendor                   ext4    ro,barrier=1,discard                                 wait,slotselect,avb,logical,first_stage_mount
 
-/dev/block/bootdevice/by-name/oem          /odm         ext4    ro,barrier=1                                                  wait,recoveryonly,slotselect
+/dev/block/bootdevice/by-name/oem_a        /odm         ext4    ro,barrier=1                                                  wait,recoveryonly,slotselect
```

Comment:
Drop the `slotselect`
Also, `recoveryonly` can go, it's a remnant of older Android times.

Reply:
Of course, I've pushed the wrong commit...